src/utils/cache.py

```diff
@@ -0,0 +1,70 @@
+"""Utility class containing instrument-specific information. This information
+is used to generate domain-valid currencies, ticker symbols, exchange codes,
+countries of origin, and any combination thereof such as RICs
+"""
+import pandas as pd
+
+
+class Cache:
+    """
+    A Class acting as a remote store for instrument-specific information.
```

Comment:
Can we remove this class and references to it in the tests? I don't see where this is used that shouldn't be replaced by the sql lite database

Reply:
ISSUE

Reply:
https://github.com/galatea-associates/fuse-test-data-gen/issues/65